geth/account/accounts.go

```diff
@@ -28,15 +28,15 @@ var (
 	ErrInvalidMasterKeyCreated         = errors.New("can not create master extended key")
 )
 
-// AccountManager represents account manager interface
-type AccountManager struct {
+// Manager represents account manager interface
+type Manager struct {
```

Comment:
Why this renaming?

Reply:
Is the `Manager` implementing a public interface (and only that)? In this case the implementation should be private and the constructor function (`NewManager()`) simply should return the interface.

Reply:
It's renamed because now it's in separate package with name `accounts`. So it's a common practice to avoid names with reduntant words like `rpc.RPCMessage` or `account.AccountManager`.

So after moving this code into separate package, we have two options to use this type:
Without renaming: `account.AccountManager`
With: `account.Manager`

I think `go lint` can spot such naming patterns and complain as well.

Reply:
Ah, yes. So it's indeed better.